data/free worlds reconciliation.txt

```diff
@@ -1640,7 +1654,6 @@ mission "FW Pug 4: Escorts"
 	source Hephaestus
 	to offer
 		has "FW Pug 3C: done"
-	
```

Comment:
Is there a reason for the whitespace change?

Reply:
Either I was going to make changes to this or was reverting changes to this made by Albert and this stuck around. There's already a mismatch in the data in terms of if there should be whitespace between the `to offer` and the rest of the mission so it's no big deal.